taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -148,17 +148,10 @@ def build_browsertime_task(config, tasks):
 @transforms.add
 def enable_webrender(config, tasks):
     for task in tasks:
-        if not task.pop("web-render-only", False):
+        if task.pop("web-render-only", False):
             newtask = copy.deepcopy(task)
             yield newtask
-        task["run"]["command"].append("--enable-webrender")
-        task["name"] += "-wr"
-        task["description"] += "-wr"
-
-        # Setup group symbol
-        group, sym = task["treeherder"]["symbol"].split("(")
-        task["treeherder"]["symbol"] = "{}-wr({})".format(group, sym[:-1])
```

Comment:
I understand the concept of removing the need for reversing this logic as we will have webrender by default.

1) will we have non webrender at all?
2) where do we add `--enable-webrender` to the command (and other stuff from the removed code)?
3) do we change something in test-sets.yml or equivalent in fenix?
4) do you have a try push or a command to run locally to generate the taskgraph?

Reply:
Thanks for the questions @jmaher as I'm more than a little unfamiliar with this code.

1. The intention is that we'll have WebRender for all tests, and non-WebRender for a subset of high value page load tests.
2. In mozilla-central this is added in a common tests transform [when the platform include "-qr"](https://searchfox.org/mozilla-central/rev/d7e344e956d9da2808ea33e1fe0f963ed10c142d/taskcluster/taskgraph/transforms/tests.py#663-666). I realise this might be entirely independent code from how we transform Fenix tasks. If so, I'll likely need to restore adding the command line option to enable WebRender. The other stuff removed here changes the Treeherder symbol, which should be uneccessary once WebRender and non-WebRender are running on different platforms.
3. Not that I'm aware of.
4. I would love to provide this, but I'm entirely uncertain of how. If you can help that would be appreciated.

Reply:
@davehunt you'll have to add the `--enable-webrender`, the fenix side doesn't do it for you like m-c does.